src/tests/JIT/interpreter/Interpreter.cs

```diff
@@ -386,4 +389,14 @@ public static bool TestVirtual()
             return false;
         return true;
     }
+
+    public static bool TestBoxing()
+    {
+        int i = 4;
+        object oI = i;
+        int j = (int)oI;
+        if (j != i)
+            return false;
+        return true;
+    }
```

Comment:
nit: from JIT viewpoint, this gets optimized to return true. We were recently using this benchmark for compulsory boxing:

```c#
using System.Runtime.CompilerServices;
...
    public object Box8() => Box(new Buffer8());
    
    [MethodImpl(MethodImplOptions.NoInlining | MethodImplOptions.NoOptimization)]
    static object Box<T>(T value) => (object)value;

    [InlineArray(1)]
    public struct Buffer8
    {
        long _element0;
    }
```